src/Makefile

```diff
@@ -48,11 +48,10 @@ GPU_SKETCH_OBJS=gpu/gpu_api.o
 CUDA_OBJS=gpu/dist.cu.o gpu/sketch.cu.o gpu/device_reads.cu.o gpu/gpu_countmin.cu.o gpu/device_memory.cu.o
 
 # web specific options
-web: CXX = emcc
+web: CXX = em++
 # optimised compile options
 # NB turn exceptions back on for testing
 web: CXXFLAGS = -O3 -s ASSERTIONS=1  \
-				--closure 1 \
```

Comment:
Why is this being removed? Does it cause difficult name mangling?
Do you plan to turn it back on again or is it not worth it?

Reply:
I just turned it off as I wasn't 100% sure how to deal with the naming when it's turned on. I thought the documentation was a bit vague, and could not find the new naming explained somewhere... But I could change that back and just use the naming Daniel used with poppunk web, if this is always the same?

Reply:
I think Daniel did a bit of grep and trial and error to find the new names... not exactly ideal I know. I think as long as the resulting JS with it off isn't really huge/non-performant it seems like it would be best to leave it off for now (but maybe leave a comment in the Makefile in case we ever want to turn it back on in future)

Reply:
I'ts weird that there is no convenient way to see how variables get renamed.
The actual difference in js filesize is 144kB vs 81bB when that flag is used. I guess the filesize ist still acceptable without the closure, so put a comment in the Makefile for now.